handbook/company/communications.md

```diff
@@ -1116,21 +1116,50 @@ As we use sentence case, only the first word is capitalized. But, if a word woul
 - When talking about a users' computer, we prefer to use "device" over _endpoint._ Devices in this context can be a physical device or virtual instance that connect to and exchange information with a computer network. Examples of devices include mobile devices, desktop computers, laptop computers, virtual machines, and servers.
 
 
-### Headings
+### Headings and titles
```

Comment:
Can we add something about headings requiring 2 extra lines of space after the previous section and 1 extra line of space between the context and the heading (e.g. lines 1112 - 1115)